File: viasoft.qualidade.rnc-frontend/apps/rnc/src/api-clients/Nao-Conformidades/Ordem-Retrabalho/model/gerar-ordem-retrabalho-validation-result.ts
export enum GerarOrdemRetrabalhoValidationResult
{
  Ok = 0,
  OperacaoFinalNaoEncontrada = 1,
  OperacaoEngenhariaDuplicada = 2,
  OdfRetrabalhoJaGerada = 3,
  LoteObrigatorio = 4,
  OdfObrigatorio = 5,
  QuantidadeInvalida = 6,
  OdfNaoApontada = 7,
  OdfNaoFinalizada = 8
}

File: viasoft.qualidade.rnc.gateway/Viasoft.Qualidade.RNC.Gateway.Host/NaoConformidades/RetrabalhoNaoConformidades/OrdemRetrabalhos/Dtos/GerarOrdemRetrabalhoValidationResult.cs
namespace Viasoft.Qualidade.RNC.Gateway.Host.NaoConformidades.RetrabalhoNaoConformidades.OrdemRetrabalhos.Dtos;

public enum GerarOrdemRetrabalhoValidationResult
{
    Ok = 0,
    OperacaoFinalNaoEncontrada = 1,
    OperacaoEngenhariaDuplicada = 2,
    OdfRetrabalhoJaGerada = 3,
    LoteObrigatorio = 4,
    LoteInexistente = 5,
    OdfObrigatorio = 6,
    OdfOrigemIsRetrabalho = 7
}